Mathlib/Algebra/Order/Ring/Lemmas.lean

```diff
@@ -271,55 +271,92 @@ alias mul_le_mul_iff_of_pos_right := mul_le_mul_right
+  (mul_le_mul_of_nonneg_right h₁ c0).trans (mul_le_mul_of_nonneg_left h₂ b0)
+
+theorem mul_lt_mul_of_le_of_ltₚ [PosMulStrictMono α] [MulPosMono α] (h₁ : a ≤ b) (h₂ : c < d)
     (a0 : 0 < a) (d0 : 0 ≤ d) : a * c < b * d :=
   (mul_lt_mul_of_pos_left h₂ a0).trans_le (mul_le_mul_of_nonneg_right h₁ d0)
-#align mul_lt_mul_of_pos_of_nonneg mul_lt_mul_of_pos_of_nonneg
+#align mul_lt_mul_of_pos_of_nonneg mul_lt_mul_of_le_of_ltₚ
 
-theorem mul_lt_mul_of_le_of_le' [PosMulStrictMono α] [MulPosMono α] (h₁ : a ≤ b) (h₂ : c < d)
-    (b0 : 0 < b) (c0 : 0 ≤ c) : a * c < b * d :=
+theorem mul_lt_mul_of_le_of_ltₚ' [PosMulStrictMono α] [MulPosMono α] (h₁ : a ≤ b) (h₂ : c < d)
+    (c0 : 0 ≤ c) (b0 : 0 < b) : a * c < b * d :=
   (mul_le_mul_of_nonneg_right h₁ c0).trans_lt (mul_lt_mul_of_pos_left h₂ b0)
-#align mul_lt_mul_of_le_of_le' mul_lt_mul_of_le_of_le'
+#align mul_lt_mul_of_le_of_le' mul_lt_mul_of_le_of_ltₚ'
 
-theorem mul_lt_mul_of_nonneg_of_pos [PosMulMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c ≤ d)
+theorem mul_lt_mul_of_lt_of_leₚ [PosMulMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c ≤ d)
     (a0 : 0 ≤ a) (d0 : 0 < d) : a * c < b * d :=
   (mul_le_mul_of_nonneg_left h₂ a0).trans_lt (mul_lt_mul_of_pos_right h₁ d0)
-#align mul_lt_mul_of_nonneg_of_pos mul_lt_mul_of_nonneg_of_pos
+#align mul_lt_mul_of_nonneg_of_pos mul_lt_mul_of_lt_of_leₚ
 
-theorem mul_lt_mul_of_le_of_lt' [PosMulMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c ≤ d)
-    (b0 : 0 ≤ b) (c0 : 0 < c) : a * c < b * d :=
+theorem mul_lt_mul_of_lt_of_leₚ' [PosMulMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c ≤ d)
+    (c0 : 0 < c) (b0 : 0 ≤ b) : a * c < b * d :=
   (mul_lt_mul_of_pos_right h₁ c0).trans_le (mul_le_mul_of_nonneg_left h₂ b0)
-#align mul_lt_mul_of_le_of_lt' mul_lt_mul_of_le_of_lt'
+#align mul_lt_mul_of_le_of_lt' mul_lt_mul_of_lt_of_leₚ'
 
-theorem mul_lt_mul_of_pos_of_pos [PosMulStrictMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c < d)
+theorem mul_lt_mul_of_lt_of_ltₚ [PosMulStrictMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c < d)
     (a0 : 0 < a) (d0 : 0 < d) : a * c < b * d :=
   (mul_lt_mul_of_pos_left h₂ a0).trans (mul_lt_mul_of_pos_right h₁ d0)
-#align mul_lt_mul_of_pos_of_pos mul_lt_mul_of_pos_of_pos
+#align mul_lt_mul_of_pos_of_pos mul_lt_mul_of_lt_of_ltₚ
 
-theorem mul_lt_mul_of_lt_of_lt' [PosMulStrictMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c < d)
-    (b0 : 0 < b) (c0 : 0 < c) : a * c < b * d :=
+theorem mul_lt_mul_of_lt_of_ltₚ' [PosMulStrictMono α] [MulPosStrictMono α] (h₁ : a < b) (h₂ : c < d)
+    (c0 : 0 < c) (b0 : 0 < b) : a * c < b * d :=
   (mul_lt_mul_of_pos_right h₁ c0).trans (mul_lt_mul_of_pos_left h₂ b0)
-#align mul_lt_mul_of_lt_of_lt' mul_lt_mul_of_lt_of_lt'
+#align mul_lt_mul_of_lt_of_lt' mul_lt_mul_of_lt_of_ltₚ'
+
+alias mul_le_mul := mul_le_mul_of_le_of_leₚ' -- this name was in `Algebra.Order.Ring.Defs`
+alias mul_lt_mul := mul_lt_mul_of_lt_of_leₚ' -- this name was in `Algebra.Order.Ring.Defs`
```

Comment:
These statements are *not* identical. `mul_lt_mul` holds in a `StrictOrderedSemiring`. Of course, the lemma you are aliasing holds in *more* generality, but it also makes Lean work harder to synthesize type classes. I am dubious whether we are actually okay with this unless I see a `!bench` showing the effect is negligible.

I've highlighted `mul_lt_mul`, but I suspect the same is true for many of the `alias`es you have in this PR.